examples/ASVideoNodeAdvanced/Sample/ViewController.m

```diff
@@ -0,0 +1,58 @@
+//
+//  ViewController.m
+//  Sample
```

Comment:
Thanks for thinking about examples!  Instead of adding a new project, would you mind adding to one of the existing ones?  In fact, it would probably be best to combine both of the Video examples into a single project (perhaps with a tab bar controller to separate the table one from the regular one).  Since pod install and building is required it's much easier for community members to check out fewer, bigger sample apps.